### PR TITLE
Release notes for Stratis 2.1.0

### PR DESCRIPTION
**Pull request by mulkieran**

### CODE REVIEW

**Comment by mulkieran** [comment]
For preliminary review.

**Comment by mulkieran** [comment]
Please let on if you think anything is missing...

File: content/2020-05-stratis.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    extended `CreatePool` method, to support an optional argument for
+    encryption. In addition, it supplies a number of method for key
+    management.
+  * `org.storage.stratis2.pool.r1`: This interface supports explicit
+    initialization of a cache tier. Previously, a cache was initialized as
+    a side-effect of the addition of the first device to the cache tier.
+    It also supports the new `Encrypted` property.
+  * `org.storage.stratis2.FetchProperties.r1`: This interface supports an
+    additional `HasCache` property.
+  * `org.storage.stratis2.Report.r1```: This interface supports a set of
```
Comment (jbaublitz) [inline content/2020-05-stratis.md:47]: This has a mismatch between one opening back-tick and three closing back-ticks.

File: content/2020-05-stratis.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    ad-hoc reports about Stratis. The interface is unstable; the names by
+    which the reports can be accesed are not guaranteed to remain stable,
+    and the format of any report is only guaranteed to be valid JSON.
+
+Please consult the D-Bus API Reference for the precise D-Bus specification.
+
+The following are significant implementation details:
+  * Each block device in an encrypted pool's data tier is encrypted with a
+    distinct, randomly chosen MEK (Media Encryption Key) on initialization.
+  * All devices belonging to a single encrypted pool share a single KEK
```
Comment (jbaublitz) [inline content/2020-05-stratis.md:57]: Passphrase, not KEK.

**Comment by mulkieran** [comment]
New subsection about usage added.

**Pull request closed without merging**